### PR TITLE
Remove use of extraneous elements. 

### PR DESCRIPTION
**Pull request by ebidel**
R: @jeffposnick @jpchase @beaufortfrancois @RByers 

A bit of code cleanup as well.

### CODE REVIEW

File: gulpfile.babel.js [hunk 1]
```diff
@@ -62,17 +62,17 @@ gulp.task('styles', () => {
...
-  // The script files are overwritten (as they are generated anyways)
-  //  - The 'base' option is needed to retain the relative path of each file,
-  //    allow overwriting
-  return gulp.src(
-    'static/elements/*.vulcanize.js', {base: 'static'}
-  )
-    .pipe(uglifyJS())
-    .pipe(gulp.dest('static'));
-});
+// // Minify the vulcanized script files
```
Comment (ebidel) [inline gulpfile.babel.js]: Don't need this task for now since the vulcanize task takes care of minifying its own js.

**Comment by jeffposnick** [comment]
You're using `fetch()` a few places here. I don't think it's the first time it's used in the codebase, but I feel like I need to point out it's [not universally supported yet](http://caniuse.com/#search=fetch). If the thinking is "It's really unlikely that anyone using the admin interface will be using Safari." then I don't have a major objection.

Alternatively, you could always pull in one of the [`fetch()` polyfills](https://github.com/github/fetch).

**Comment by jeffposnick** [comment]
Other than the `fetch()` comment, 👍

**Comment by ebidel** [comment]
Exactly, my thinking was that anyone hitting the /admin pages is a chromium/blink user and therefore, `fetch()` is good to go!

**Pull request merged**